### ISSUE TITLE
Possible 1.4.1 failure in WCAG 2.1 normative document

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wcag#1048]
If you look at the WCAG 2.1 document today, above the abstract, there are a few links that do not have an underline. Some of them are bold, but one is not, making it only distinguishable from the surrounding text by its blue color. This seems to me like it would be a failure of SC 1.4.1 Use of Color. The issue is there on three links:

- "errata" in "Please check the errata for any errors ..."
- "translation" in "See also translations."
- "Alternate Versions of Web Content Accessibility Guidelines 2.1"

![Screenshot 2020-02-25 13 04 34](https://user-images.githubusercontent.com/530687/75246213-9fdb0800-57cf-11ea-83a0-90b9f996f185.png)

Credit to Eric Velleman at Accessibility Foundation for spotting this.

**Comment by awkawk** [comment]
I believe that we have an issue already defined for this, on Michael’s plate...

**Comment by awkawk** [comment]
@alastc filed the #562 issue only on the third link. The Errata and Translations links arguably pass because they are bold and differentiate from adjacent text by color AND font-weight. That's a finer point than I'd like to rely on, but this is in the W3C spec template, so is a bit more difficult to fix as a result.

@michael-n-cooper Can you make sure that at a minimum we make the link bold in the generated page?

**Comment by alastc** [comment]
Getting to know the setup better, I think we could add this to the WCAG styles:

```
.head p:not(.copyright) > a, .head > a:first-child {
   border-bottom: 1px solid #707070;
}
```

That would be in keeping with the other links in the area.

I've created a PR for that, if Michael's happy I think we can merge and close these.

**Comment by alastc** [comment]
I've just noticed that none of the [recent specs](https://www.w3.org/TR/) are missing underlines in the header area, is our CSS behind?

**Comment by michael-n-cooper** [comment]
This is caused by the central TR design and we can't override it. I filed https://github.com/w3c/tr-design/pull/187 to address this, though it's likely to be a long time before we see results.

**Comment by fantasai** [comment]
Fixed in the tr-design sources. I'm not sure what the update schedule is, but you can ask your staff contact maybe they know...

**Comment by alastc** [comment]
Thanks @fantasai, that would be @michael-n-cooper so I will pester him now :-)
